README.md

```diff
@@ -23,14 +23,14 @@ Input Example with `OK Button Caption`(Optional):
    input message: 'input-message', ok: 'OK'
 
 Input Example with `Allowed Submitter`:
-User IDs and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
+User ids and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
```

Comment:
In this case, I think that many places in the Jenkins documentation and changelogs use the word "username" to refere to the identifier of a user.
```suggestion
Usernames and/or external group names of those permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
```

Reply:
done